rspace/src/main/scala/coop/rchain/rspace/state/exporters/RSpaceExporterItems.scala

```diff
@@ -62,4 +55,32 @@ object RSpaceExporterItems {
       keys     = dataKeys.distinct.map(_.hash)
       items    <- time("GET DATA BY KEYS")(exporter.getDataItems(keys, fromBuffer))
     } yield StoreItems(items, lastEntry.path :+ (lastEntry.hash, none))
+
+  def getHistoryAndData[F[_]: Sync, Value](
+      exporter: RSpaceExporter[F],
+      startPath: Seq[(Blake2b256Hash, Option[Byte])],
+      skip: Int,
+      take: Int,
+      fromBuffer: ByteBuffer => Value
+  ): F[(StoreItems[Blake2b256Hash, Value], StoreItems[Blake2b256Hash, Value])] =
+    for {
+      // Traverse and collect tuple space nodes
+      nodes <- time("TRAVERSE TRIE")(exporter.getNodes(startPath, skip, take))
```

Comment:
Just out of curiosity for now - is using this `time` in production expected? Or this should be replaced at some point with Jaeger spans?

Reply:
It's temporary, should be removed before merge to dev.